### DISCUSSION TITLE
Data import from Youtube Music

### DISCUSSION THREAD

**Discussion by koiskarp** [KRTirtho/spotube#1731] [Ideas]
Hi there,
   As for now, I am a user who plans to turn to Spotube from Youtube Music. I find your application quite convenient from the point of complete abscence of adverts: now I can finally listen to music without ear-breakening interruptions from the adverts of Pepsi)
   To make the application even better, I would like to suggest adding an option to import user preferences (albums, tracks and so on) from YT Music, which can make the transfer for new users more convenient: they will not require building up their playlists again.
   Thanks a lot for a review and have a good day)

**Comment by LetsBuild-exe** [comment]
That propably wont be possible cause it interface ur playlist from spotify, there are services to port from youtube music to spotify u should find some if u search a bit
